.env

```diff
@@ -0,0 +1,5 @@
+# S3 testing related data
+S3_BUCKET="locals3"
+S3_REGION="us-east-1"
+AWS_ACCESS_KEY_ID="test"
+AWS_SECRET_ACCESS_KEY="test"
```

Comment:
could you document in the readme how a local development setup would use these .env files?

Reply:
I uploaded a `.env.template` copy of the `.env` and added the fields to the docker-compose instance alongside a Readme explanation.

Reply:
I uploaded a `.env.template` copy of the `.env` and added the fields to the docker-compose instance alongside a Readme explanation.